README.md

```diff
@@ -64,7 +64,7 @@ See [here](https://docs.immutable.com/docs/x/passport/register-application/) for
 
 > [!NOTE]
 > For Unreal Engine 4.26 and above we use Blui as a browser instead of inbuilt browser.
```

Comment:
Can this be changed to say Unreal Engine 4.x. Currently it implies >= 4.26 which includes UE5.

Reply:
Sorry, I meant to write UE4

Reply:
Please see the 3rd point below. However might be good to put that all in one line like

> For Unreal Engine 4.26 and above we use Blui as a browser instead of inbuilt browser. For Unreal Engine 5.0 and above we use inbuilt browser